modules/grpc/otel/otel-source.cpp

```diff
@@ -49,11 +49,8 @@ syslogng::grpc::otel::SourceDriver::SourceDriver(OtelSourceDriver *s)
 void
 syslogng::grpc::otel::SourceDriver::request_exit()
 {
-  if (server == nullptr)
-    return;
-
-  server->Shutdown();
-  server = nullptr;
```

Comment:
My understanding is that `server` (`unique_ptr`) gets destroyed together with the SourceDriver now.